pkg/kubectl/kubectl.go

```diff
@@ -34,13 +34,73 @@ type kubeCtlVersionData struct {
 	ServerVersion *kubeCtlVersionInfo `json:"serverVersion,omitempty"`
 }
 
+// Example stdout:
+// Client Version: version.Info{Major:"1", Minor:"9", GitVersion:"v1.9.2", ..., Platform:"linux/amd64"}
+// Server Version: version.Info{Major:"1", Minor:"9", GitVersion:"v1.9.3", ..., Platform:"linux/amd64"}
+
+// We don't care about the exact reason why the parsing failed, we'll display
+// more context in the error message anyway.
+var errParsing = errors.New("parse error")
+
+func parseVersionLine(line string, versionInfo **kubeCtlVersionInfo) error {
```

Comment:
How closely does this match what launch-generator is doing?

Reply:
regex ftw! https://github.com/weaveworks/launch-generator/blob/master/src/utilities/kubectl.js#L7